app/js/containers/Buyer/Listing.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../../node_modules/bootstrap/scss/functions";
+@import "../../../../node_modules/bootstrap/scss/variables";
+
+$logo-size: 64px;
+.logo-circle {
+  width: $logo-size;
+  height: $logo-size;
+  line-height: $logo-size;
+  font-size: 28px;
+  color: white;
+  font-weight: bold;
+}
+
+.offer-listing {
+  color: $yiq-text-dark;
+
+  .rating-container {
+    span {
+      border-top-right-radius: 15px;
+      border-bottom-right-radius: 15px;
+    }
+
+    span:first-child {
+      border-radius: 15px 0 0 15px;
+    }
+  }
+}
+
+.sort-icon {
```

Comment:
If it is not wrapped around container name I feel this need to be extracted as a component